src/components/10-atoms/radio/index.scss

```diff
@@ -0,0 +1,190 @@
+  flex: none;
+
+  *, *::after, *::before {
+    box-sizing: border-box;
+  }
+
+  &:hover:not([disabled]):not(.a-radio--error) {
+    color: $color-axa-blue;
+
+    & .a-radio__icon {
+      border-color: $color-axa-blue;
+
+      &::after {
+        border-color: $color-axa-blue;
+      }
+    }
+  }
+
+  .a-radio__wrapper {
+    position: relative;
+    display: -webkit-inline-box;
+    display: -ms-inline-flexbox;
+    display: inline-flex;
+    min-height: 20px;
+    -webkit-box-align: center;
+        -ms-flex-align: center;
+            align-items: center;
+    line-height: 1.5;
+    justify-content: center;
+  }
+
+  .a-radio__input {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    clip: rect(0, 0, 0, 0);
+    margin: -1px;
+    overflow: hidden;
+
+    &:focus + .a-radio__icon {
+      border-width: 2px;
+      border-color: $color-axa-blue;
+    }
+
+    &:checked + .a-radio__icon::after {
+      border-color: $color-prim-gray-dusty;
+      background-color: $color-axa-blue;
+    }
+
+    &:disabled + .a-radio__icon {
+      cursor: default;
+      border-color: $color-prim-gray-silver;
+
+      &::after {
+        border-color: $color-prim-gray-silver;
+        background-color: $color-prim-gray-silver;
+      }
+    }
+  }
+
+  .a-radio__icon {
+    -webkit-transition: all 0.2s ease;
+    transition: all 0.2s ease;
+    width: 18px;
+    height: 18px;
+    display: -webkit-inline-box;
+    display: -ms-inline-flexbox;
+    display: inline-flex;
+    margin-right: 20px;
+    -webkit-box-pack: center;
+        -ms-flex-pack: center;
+            justify-content: center;
+    -webkit-box-align: center;
+        -ms-flex-align: center;
+            align-items: center;
+    background: #fff;
+    border: 1px solid $color-prim-gray-dusty;
+    cursor: pointer;
+    -ms-flex-negative: 0;
+        flex-shrink: 0;
+    border-radius: 50%;
+    box-sizing: border-box;
+
+    &::after {
+      width: 8px;
+      height: 8px;
+      display: block;
+      border: 1px solid transparent;
+      content: "";
+      border-radius: 50%;
+      box-sizing: border-box;
+    }
+  }
+
+  .a-radio__content {
+      display: block;
+      margin: 0;
+      line-height: 1.37;
+      color: $color-prim-gray-mine-shaft;
+  }
+
+  &[button] {
+    position: relative;
+    display: -webkit-inline-box;
+    display: -ms-inline-flexbox;
+    display: inline-flex;
+    height: 40px;
+
+    .a-radio__wrapper {
+      background-color: $color-prim-gray-wild-sand;
+      border: 1px solid $color-prim-gray-dusty;
+      padding: 0 20px;
+      cursor: pointer;
+      width: inherit;
+
+      &:hover {
+        border-color: $color-axa-blue;
+        color: $color-axa-blue;
+      }
+    }
+
+    .a-radio__content {
+        display: table;
+        background-color: $color-prim-gray-wild-sand;
+        padding: 0;
+        text-align: center;
+        text-transform: uppercase;
+        line-height: 18px;
+        letter-spacing: 0.56px;
```

Comment:
0.56px can maybe be dangerous as some browser might round up or down pixels. Now i dont know exactly how it is with letter spacing, just read my comment and verify if there is a risk of having browsers that round down to 0px. if im over-worrying, u can ignore this comment

Reply:
Again this was literally taken from Zeplin.

Reply:
Supported by browsers since 2013: https://css-tricks.com/almanac/properties/l/letter-spacing/.

And CSS 2 Spec says letter-spacing is _in addition to_ default spacing.